### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Comment by VincentLanglet** [comment]
Need https://github.com/sonata-project/SonataAdminBundle/pull/6829 first

**Comment by VincentLanglet** [comment]
```
<!-- An exception has been thrown during the rendering of a template (&quot;The method &quot;name&quot;() does not exist.&quot;). (500 Internal Server Error) -->
```

**Comment by VincentLanglet** [comment]
This is ready to merge @sonata-project/contributors

**Review by dmaicher — APPROVED** [review]

**Review by franmomu — APPROVED** [review]

File: src/Builder/DatagridBuilder.php [hunk 1]
```diff
@@ -135,7 +135,6 @@ public function fixFieldDescription(AdminInterface $admin, FieldDescriptionInter
         //    $fieldDescription->setOption('parent_association_mappings', $fieldDescription->getOption('parent_association_mappings', $fieldDescription->getParentAssociationMappings()));
         //}
 
-        $fieldDescription->setOption('code', $fieldDescription->getOption('code', $fieldDescription->getName()));
```
Comment (franmomu) [inline src/Builder/DatagridBuilder.php:138]: why this change?

Comment (VincentLanglet) [reply src/Builder/DatagridBuilder.php:138]: Cause now, passing a non working `code` throw an exception:
https://github.com/sonata-project/SonataAdminBundle/blob/master/src/Admin/BaseFieldDescription.php#L404-L430
